src/components/tables/Collections/Rows.tsx

```diff
@@ -41,11 +57,33 @@ export const tableColumns = [
     },
 ];
 
-function Row({ row, showEntityStatus }: RowProps) {
+function Row({ row, stats, showEntityStatus }: RowProps) {
     const theme = useTheme();
 
     const [detailsExpanded, setDetailsExpanded] = useState(false);
 
+    const calculatedBytes = useMemo(() => {
+        if (stats) {
+            return (
+                (stats[row.catalog_name]?.bytes_written_by_me ?? 0) +
+                (stats[row.catalog_name]?.bytes_written_to_me ?? 0)
+            );
+        } else {
+            return 0;
+        }
+    }, [row.catalog_name, stats]);
+
+    const calculatedDocs = useMemo(() => {
+        if (stats) {
+            return (
+                (stats[row.catalog_name]?.docs_written_by_me ?? 0) +
+                (stats[row.catalog_name]?.docs_written_to_me ?? 0)
+            );
+        } else {
+            return 0;
+        }
+    }, [row.catalog_name, stats]);
```

Comment:
Collection needs to have these in a memo because it is just easier to write the calculations like this instead of inside nested ternaries down below.